.travis.yml

```diff
@@ -10,13 +10,53 @@ android:
   - android-26
 env:
 - GRADLE_OPTS="-XX:MaxPermSize=2048M"
+stages:
+  - name: test
+  - name: deploy
+    if: tag IS present
 jobs:
   include:
-  - script: make test
-  - script: make travis
-    if: tag IS present
+    - stage: test
+      script: make test
+  
+    - stage: deploy
+    # flavors sorted alphabetically with three per job
+      script:
+      - fastlane deploy flavor:unbranded
+      - fastlane deploy flavor:medicmobiledemo
+      - fastlane deploy flavor:bracuganda
+    - script:
+      - fastlane deploy flavor:cic_guatemala
+      - fastlane deploy flavor:ebpp_indonesia
+      - fastlane deploy flavor:hope_through_health
+    - script:
+      - fastlane deploy flavor:livinggoods
+      - fastlane deploy flavor:livinggoodskenya
+      - fastlane deploy flavor:livinggoods_assisted_networks
+    - script:
+      - fastlane deploy flavor:livinggoods_innovation_ke
+      - fastlane deploy flavor:livinggoods_innovation_ke_supervisor 
+      - fastlane deploy flavor:livinggoods_innovation_ke_hivst
+    - script:
+      # - fastlane deploy flavor:moh_kenya_siaya_white
+      - fastlane deploy flavor:moh_kenya_siaya_red
+      - fastlane deploy flavor:moh_kenya_siaya_green
+    - script:
+      - fastlane deploy flavor:moh_kenya_siaya_black
+      - fastlane deploy flavor:moh_zanzibar_training
+      - fastlane deploy flavor:moh_zanzibar
+    - script:
+      - fastlane deploy flavor:musomali
+      - fastlane deploy flavor:pih_malawi
+      - fastlane deploy flavor:pih_malawi_supervisor
+    - script:
+      - fastlane deploy flavor:simprints
+      - fastlane deploy flavor:vhw_burundi
```

Comment:
It'd be awesome to generate this list dynamically but I can't see any way to do it in yaml. We could write a script to generate it precommit or something but that's probably a bit messy. This is good for now.

Reply:
Thought about that also. If it were 50 or if the list changed frequently, I'd have done it. But since it is so short ...